src/lib/components/ui/Search.svelte

```
@@ -0,0 +1,206 @@
+	const client = algoliasearch('99IEWD8Z0K', 'c6f824db1a70a8523780908459090a48');
+
+	type SearchResult = {
+		objectID: string;
+		title: string;
+		_highlightResult?: { [key: string]: HighlightResult } | undefined;
+		_snippetResult?: { [key: string]: SnippetResult } | undefined;
+		_rankingInfo?: RankingInfo | undefined;
+		_distinctSeqID?: number | undefined;
+	};
+
+	const results = writable<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)